vultr.php

```diff
@@ -29,6 +29,259 @@ public function __construct()
         $this->loadConfig(dirname(__FILE__) . DS . 'config.json');
     }
 
+    /**
+     * Performs migration of data from $current_version (the current installed version)
+     * to the given file set version. Sets Input errors on failure, preventing
+     * the module from being upgraded.
+     *
+     * @param string $current_version The current installed version of this module
+     */
+    public function upgrade($current_version)
+    {
+        // Upgrade to 2.0.0
+        if (version_compare($current_version, '2.0.0', '<')) {
```

Comment:
I tried creating a service on the previous version, then updating. That service was not properly updated.  It was a VM, not a baremetal.

Reply:
There was a specific error or unexpected behavior?

Reply:
Sorry, I was vague.  The subid service field was not updated and thus none of the management tabs or api calls functioned.

Reply:
I was not able to reproduce the issue, but I made some updates to improve the upgrade process.